CONTRIBUTING.md

```diff
@@ -9,7 +9,12 @@ Suport for clang-format is built-in since Visual Studio 2017 15.7 and CLion 2019
 The .clang-format file in the repository will be automatically detected and formatting is done as you type, or triggered when pressing the format hotkey.
 
 ### Bash
-First install clang-format. Instructions therefore can be found on the web. To format you can run this command in bash:
+First install clang-format. Instructions therefore can be found on the web.
+To format your changes since branching off develop, you can run this command in bash:
 ```
-find -iname *.cu -o -iname *.hpp | xargs clang-format-10 -i
+git clang-format develop
```

Comment:
should this not be `git clang-format  dev` or is the last parameter not the branch?

Reply:
The last parameter is the branch name. I was confused because the `dev` branch is called `develop` in alpaka and llama. Should we rename the `dev` branch to `develop` in mallocMC?

Reply:
in PIConGPU the branch is also called dev. alpaka is the only project which uses a different name

Reply:
could you update this pr to mention dev in this command

Reply:
I changed the PR to name the branch `dev`.